src/components/timetable/view/TimetableSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Moment } from 'moment';
+
+import TimePlan from '@/components/plan/TimePlan';
+import TimePlanManager from '@/plan/TimePlanManager';
+import useFocusedPlanState from '@/stores/plan/focusedPlan';
+
+type TProps = {
+  dateMoment: Moment;
+};
+
+const TimetableSelect: React.FC<TProps> = ({ dateMoment }) => {
+  const { focusedPlan } = useFocusedPlanState();
```

Comment:
```suggestion
  const focusedPlan = useFocusedPlanState(store => store.focusedPlan);
```
으로 하면 리렌더링이 덜 일어날 것 같아요

Reply:
Zustand의 리렌더링 관련 코드 변경은 드래그 기능 완성 후 이슈 작성으로 변경하겠습니다!!